webapp/src/main/resources/public/js/stores/workbench/GitBlameScreenshotViewerStore.js

```diff
@@ -0,0 +1,54 @@
+import alt from "../../alt";
+import GitBlameScreenshotViewerActions from "../../actions/workbench/GitBlameScreenshotViewerActions";
+
+class GitBlameScreenshotViewerStore {
```

Comment:
it doesnt make sense to extend the BranchesScreenshotViewerStore, they dont share code

Reply:
I've check the code more and yes it is almost the same thing. The only difference at the moment is the screenshot list that is passed instead of the `branchStatisticId`.

The base store can be similar to this one and `BranchesScreenshotViewerStore` changed with the open function be something like 
```javascript
open(branchStatisticId) {
    let branchStatisticScreenshots = BranchesStore.getBranchStatisticById(branchStatisticId).branch.screenshots
    super.open(branchStatisticScreenshots);
}
```